runtime/test_artifacts/defaults.go

```diff
@@ -8,7 +8,7 @@ import (
 
 var (
 	DefaultChains              = []string{"0001"}
-	DefaultServiceURL          = ""
+	DefaultServiceURL          = "https://localhost.com"
```

Comment:
Not sure about this value here. Below, in `ServiceUrlFormat` we define it as `(host):(port)`

Reply:
Reverting.